File: 1828.py
a = int(input())
lista = []
vezes = 0
for i in range(0, a):
    sheldon, raj = input().split()
    if sheldon == 'tesoura' and raj == 'papel' or sheldon == 'papel' and raj == 'pedra' or sheldon == 'pedra' and raj == 'lagarto' or sheldon == 'lagarto' and raj == 'Spock' or sheldon == 'Spock' and raj == 'tesoura' or sheldon == 'tesoura' and raj == 'lagarto' or sheldon == 'lagarto' and raj == 'papel' or sheldon == 'papel' and raj == 'Spock' or sheldon == 'Spock' and raj == 'pedra' or sheldon == 'pedra' and raj == 'tesoura':
        lista += ['Bazinga!']
        vezes += 1
    if raj == 'tesoura' and sheldon == 'papel' or raj == 'papel' and sheldon == 'pedra' or raj == 'pedra' and sheldon == 'lagarto' or raj == 'lagarto' and sheldon == 'Spock' or raj == 'Spock' and sheldon == 'tesoura' or raj == 'tesoura' and sheldon == 'lagarto' or raj == 'lagarto' and sheldon == 'papel' or raj == 'papel' and sheldon == 'Spock' or raj == 'Spock' and sheldon == 'pedra' or raj == 'pedra' and sheldon == 'tesoura':
        lista += ['Raj trapaceou!']
        vezes += 1
    if sheldon == raj:
        lista += ['De novo!']
        vezes += 1

for j in range(0, vezes):
    print(f'Caso #{j+1}: {lista[j]}')

File: 1828.cpp
#include <iostream>
#include <algorithm>
#include <string>
using namespace std;

int main(){
    int numero_casos, contador = 0;
    cin >> numero_casos;
    string sheldon, raj;
    for(int i = 0; i < numero_casos; i++){
        cin >> sheldon >> raj;
        if(sheldon == raj){
            contador += 1;
            cout << "Caso #" << contador << ": De novo!\n";
        }else if(sheldon == "tesoura" && raj == "papel" || sheldon == "papel" && raj == "pedra" || sheldon == "pedra" && raj == "lagarto" || sheldon == "lagarto" && raj == "Spock" || sheldon == "Spock" && raj == "tesoura" || sheldon == "tesoura" && raj == "lagarto" || sheldon == "lagarto" && raj == "papel" || sheldon == "papel" && raj == "Spock" || sheldon == "Spock" && raj == "pedra" || sheldon == "pedra" && raj == "tesoura"){
            contador += 1;
            cout << "Caso #" << contador << ": Bazinga!\n";
        }else{
            contador += 1;
            cout << "Caso #" << contador << ": Raj trapaceou!\n";
        }
    }
    return 0;
}
